Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/Raw/BoundarySegmentConversionToSpeckle.cs

```diff
@@ -28,9 +29,10 @@ public SOG.Polycurve Convert(IList<DB.BoundarySegment> target)
       // POC: We used to attach the `elementID` of every curve in a PolyCurve as a dynamic property.
       // We've removed this as it seemed unnecessary.
 
-      poly.segments.Add(curve);
+      segments.Add(curve);
     }
 
+    var poly = new SOG.Polycurve { segments = segments, units = Units.Meters }; //TODO: is the units here correct?
```

Comment:
should be `_contextStack.Current.SpeckleUnits` i suspect

Reply:
done!

Reply:
wait, this is still Units.Meters

Reply:
good catch, I forgot to push, should be correct now